lab6.md

```diff
@@ -123,6 +123,36 @@ $ faas-cli up -f show-html.yml
 
 Open your browser and access http://127.0.0.1:8080/function/show-html. You should see a "Here's a new page!" HTML page rendered in the browser.
 
+Now try how it works with a path.
+
+Edit your `handler.py` to the following:
+
+```python
+import os
+
+def handle(req):
+    
+    path = os.environ['Http_Path']
+    pathArr = path.split("/")
+    pageName = pathArr[1]
+    
+    dirname = os.path.dirname(__file__)
+    page = os.path.join(dirname, 'html', pageName + '.html')
+
+    with(open(page, 'r')) as file:
+        html = file.read()
+
+    return html
+```
+
+Build, push and deploy the function:
+
+```
+$ faas-cli up -f show-html.yml
+```
+
+Now open your web page by http://127.0.0.1:8080/function/show-html/new or http://127.0.0.1:8080/function/show-html/list.
```

Comment:
Maybe give them the text for list.html and new.html? 2-3 lines of HTML with a H2 would be fine.